cookbooks/cdo-apps/libraries/cdo_apps.rb

```diff
@@ -104,4 +104,28 @@ def setup_app(app_name)
       notifies :restart, "service[#{app_name}]", :immediately
     end
   end
+
+  def setup_build(package)
+    include_recipe 'cdo-nodejs'
+    user = node[:user]
+    home = node[:home]
+    root = File.join home, node.chef_environment
+
+    utf8 = 'en_US.UTF-8'
+    env = {
+      'LC_ALL' => utf8,
+      'LANGUAGE' => utf8,
+      'LANG' => utf8,
+      'RAILS_ENV' => node.chef_environment
+    }
+    execute "build:#{package}" do
+      command "bundle exec rake build:#{package}"
+      cwd root
+      environment env.merge(node['cdo-apps']['bundle_env'])
+      user user
+      group user
+      not_if { File.directory? "#{root}/#{package}/node_modules"}
```

Comment:
oh, good catch! Do you have any preference on adding the Rakefile aliases for the dash keys (as I've already done with blockly-core), versus just explicitly mapping the underscore folder to dashed versions of rake tasks within this function (e.g., via `package.tr('_', '-')`)?